Lab 5

XomsEdu opened the pull request.

XomsEdu committed:
Pre-lab 5 pull request

XomsEdu committed:
Lab 5

Фікси після 4ї лаби і база для модалки 5ї лаби

vozniuk-lntu reviewed (changes requested):
1. ховер на кнопку закрити модалку
2. не зробив https://i.imgur.com/dXbVMO4.png
3. не зробив https://i.imgur.com/Djd2Iph.png
4. .social_networks забрати падінги. member має мати падінги 30пх

XomsEdu committed:
Fixes Lab-5

Додав ховер на хрест в модалці, зробив вспливашки в портфоліо і текст поверх фоток на головній (там є певні траби з полосками які я не розібрався, можливо баг лайвсервера), пофіксив падінги але не переніс їх в .member а пофіксив їх в social networks.
